### ISSUE TITLE
Move banner into <main>

### ISSUE THREAD

**Issue by seanredmond** [NYPL-Simplified/web-reader-landing-page#26]
In index.html.template, `div.banner` should be a child of the `<main>` element. Then everything is part of an aria landmark. Otherwise a screen reader user who jumped to the main landmark would miss the description.
